template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -77,6 +77,13 @@ def get_project_name():
 
 
 def get_args():
+    def yes_no_auto(value):
```

Comment:
Actually this is unnecessary. If litani supports pools then auto == yes, and if it doesn't then auto, yes, and no == no.

So the behaviour should just be: defaut is "use pools iff litani supports it". There should also be a flag `--no-expensive-pool` that disables pools if litani supports it (and is a noop otherwise).